practices/styleguide.md

```diff
@@ -0,0 +1,13 @@
+# Styleguide
```

Comment:
```suggestion
# Style guide for repositories
```

Just to clarify a bit (also I _think_ style guide is two words but I am not positive haha)